test/unittests/execute_floating_point_test.cpp

```diff
@@ -440,6 +492,72 @@ TYPED_TEST(execute_floating_point_types, neg)
     }
 }
 
+TYPED_TEST(execute_floating_point_types, ceil)
+{
+    auto instance = instantiate(parse(this->get_unop_code(Instr::f32_ceil)));
+    const auto exec = [&](auto arg) { return execute(*instance, 0, {arg}); };
+
+    // fceil(-q) = -0  (if -1 < -q < 0)
```

Comment:
I'm not a fan of duplicating these cases, it might be confusing why it's tested two times.

Perhaps a comment, that it's tested in the loop over `positive_trunc_tests`, but added here for explicitness, would help.

Reply:
These are not tested two times, these are some edge cases.

Reply:
These are the same cases as `positive_trunc_tests` items 
`{std::nextafter(T{1}, T{0}), T{0}}` 
`{L::denorm_min(), 0}`  tested in the loop as
```
EXPECT_THAT(exec(-arg), Result(-expected_trunc)) << -arg << ": " << -expected_trunc;
```

similar in tests for other instructions

Reply:
Are we fine here?

Reply:
@gumb0 you are correct. I still prefer if we have clear single test cases for the edge cases listed in the specific for the given instruction. It is much easier to review that every case is tested.

Stuff hidden in those loops is extremely hard to review, in my opinion.

Reply:
I'm fine with including it, already approved.